### PR TITLE
WIP: Update Chapter06.md

### PR DESCRIPTION
**Pull request by pgoyal01**
Related to Issue #149 -- added technology agnostic hardware acceleration interfaces as defined by ETSI IFA 019 v03101p

### CODE REVIEW

File: doc/ref_model/chapters/chapter06.md [hunk 1]
```diff
@@ -104,9 +104,48 @@ A virtual compute resource is created as per the flavour template (specifies the
...
 This is a place holder for Enabler Services APIs.
 a.	NTP, DNS, etc. – where is the care and feeding of these? Who provides certain features/services within or outside the tenant?
-b.	Licensing and imaging connectivity
\ No newline at end of file
+b.	Licensing and imaging connectivity
+
+<a name="6.4"></a>
+## 6.4 Hardware Acceleration Interfaces and APIs 
+
+**Virtual GPU**: A physical Graphics Processing Unit (pGPU) can be virtualized as multiple virtual Graphics Processing Units (vGPUs) if the hypervisor supports the hardware driver and has the capability to create guests using those virtual devices. The virtual GPU feature in some VIMs allows a deployment to provide specific GPU types for instances using physical GPUs that can provide virtual devices. The steps required to enable virtual GPUs include (i) enabling GPU types; and (ii) configuring an instance type to use a vGPU. Comoute hosts properly configured for vGPU support then pmake it ossible to create instances (VMs) with virtual GPU devices.
```
Comment (CsatariGergely) [inline doc/ref_model/chapters/chapter06.md]: nit: pmake it ossible -> make it possible

**Comment by rabi-abdel** [comment]
@pgoyal01 we need more clarity about the hardware acceleration interfaces. are you proposing  a single interface for all acceleration types? IFA19 has set some requirements for such interfaces but IFA002 has specified those interfaces.

need some clarity about if the details here are meant to be requirement for hardware acceleration interfaces or specs.

**Comment by pgoyal01** [comment]
@rabiabdel @walterkozlowski -- can you please reveiw? Thanks

**Comment by rabiabdel** [comment]
More work is needed.

**Comment by pierrebichon** [comment]
test

**Comment by rabiabdel** [comment]
Passed 48 hours, will be approved online.

**Pull request merged**